orttraining/tools/ci_test/download_e2e_test_data.py

```diff
@@ -11,8 +11,8 @@
 import zipfile
 
 # update these if the E2E test data changes
-ARCHIVE_BLOB_URL = "https://onnxruntimetestdata.blob.core.windows.net/training/onnxruntime_training_data.zip?snapshot=2020-04-14T02:10:05.3158245Z"
-ARCHIVE_SHA256_DIGEST = "ea4168a801ded478f4e2af08232cb1174913caac300d5bf73b2652dc6894372c"
+ARCHIVE_BLOB_URL = "https://onnxruntimetestdata.blob.core.windows.net/training/onnxruntime_training_data_v12.zip?snapshot=2020-06-13T06:24:15.0833240Z"
```

Comment:
>onnxruntime_training_data_v12 [](start = 79, length = 29)

do we need a new one or can we use the same archive name?

Reply:
I prefer new one is there's concern, we can deprecate old one later

Reply:
i don't see why we need two currently - the archive is versioned by the snapshots. if we will run two separate sets of tests which require different versions, keeping them separate makes sense.

---
In reply to: [440456503](https://github.com/microsoft/onnxruntime/pull/4238#discussion_r440456503) [](ancestors = 440456503)

Reply:
OK, let me work on it